lib/decidim/calendar/version.rb

```diff
@@ -3,10 +3,10 @@
 module Decidim
   # Holds decidim-calendar version
   module Calendar
-    DECIDIM_VERSION = [">= 0.28.0", "< 0.29"].freeze
+    DECIDIM_VERSION = [">= 0.29.2", "< 0.30"].freeze
 
     def self.version
-      "0.28.0"
+      "0.29.0"
```

Comment:
```suggestion
    0.29
```
Same as previous, better use the minimal version for 0.29 here.